src/lib.pyx

```diff
@@ -0,0 +1,261 @@
+from enum import IntEnum, auto
+
+cdef extern from "slvs.h":
+    ctypedef int Slvs_hEntity
+    ctypedef int Slvs_hGroup
+    ctypedef int Slvs_hConstraint
+    ctypedef int Slvs_hParam
+    ctypedef struct Slvs_SolveResult:
+        int result
```

Comment:
I am not happy with `SolveResult.result`, should probably look for a better name here